WooCommerce/Classes/Authentication/Store Creation/Plan/StoreCreationPlanFeaturesView.swift

```diff
@@ -5,13 +5,18 @@ struct StoreCreationPlanFeaturesView: View {
     /// Features to show in a vertical list.
     let features: [StoreCreationPlanViewModel.Feature]
 
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
     var body: some View {
         VStack(alignment: .leading, spacing: 16) {
             ForEach(features, id: \.title) { feature in
                 HStack(spacing: 12) {
                     Image(uiImage: feature.icon)
                         .renderingMode(.template)
                         .foregroundColor(Color(.wooCommercePurple(.shade90)))
+                        .scaledToFit()
+                        .frame(width: 18 * scale, height: 18 * scale)
```

Comment:
I believe `resizable()` is needed when we explicitly set a frame to `Image`, that way the content can be resized.
| Without `resizable` | With `resizable` |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/204967877-1e4079ba-0cb3-43e3-8adf-c1da42910cef.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/204967890-81ee62e1-ba93-42f7-91a0-1b1bb892fb28.png" width=320 />

Reply:
Should we scale that image for WP.com as well? ☝️

Reply:
> I believe `resizable()` is needed when we explicitly set a frame to `Image`, that way the content can be resized.
> 
> Without `resizable`	With `resizable`

Great catch, updated in https://github.com/woocommerce/woocommerce-ios/pull/8271/commits/409da051de9bdd04f48d4f95cb4b4d5fb0c33be4

> Should we scale that image for WP.com as well? ☝️

Good idea, updated in https://github.com/woocommerce/woocommerce-ios/pull/8271/commits/d14c44a189abe0a311bf593d8c00bcb81adc51f8